README.md

```diff
@@ -93,311 +102,28 @@ run it with its built-in emulator.
-plugin named "foobar" that is listening on 10.10.1.8:5001, the environment variable would
-look like:
-```
-SYNSE_PLUGIN_TCP_FOOBAR = 10.10.1.8:5001
-```
-
-#### Unix Socket
-If a plugin is configured for Unix sockets in Synse Server, it should also be configured for Unix
-sockets in the plugin configuration. See the [SDK documentation][synse-sdk] for more info on plugin
-configuration.
-
-There are three ways of configuring a Unix socket plugin with Synse Server:
- 1. By config file
- 2. By environment variable
- 3. By using the default socket path
- 
-To configure **by config file**, you will need to specify the name of the plugin and the 
-directory where the socket exists. The socket is expected to have the same name as the plugin.
-This is specified under the `plugin.unix` field of the configuration. As an example,
-for a plugin named "foobar" whose socket, "foobar.sock", is in the directory "/tmp/run", the
-configuration would look like
-```yaml
-plugin:
-  unix:
-    foobar: /tmp/run
-```
-
-To configure **by environment variable**, you will need to specify an environment variable
-with the `SYNSE_` prefix followed by the configuration path, followed by the plugin name. The
-value of the environment variable should be the directory containing the socket for the plugin.
-As an example, for a plugin named "foobar" whose socket, "foobar.sock", is in the directory 
-"/tmp/run", the environment variable would look like:
-```
-SYNSE_PLUGIN_UNIX_FOOBAR = /tmp/run
-```
-
-To configure **by using the default socket path**, then all you will have to do is put the socket
-file into the default socket path for Synse Server (Note: if running Synse Server in a container,
-this means volume mounting). The default path is `/tmp/synse/procs`, so if you had a plugin named
-"foobar" whose socket, "foobar.sock" was in "/tmp/synse/procs", then you would not need to specify
-anything in the YAML configuration or via environment variable. 
-
-While its not necessary to specify anything in this case, it is often still good practice to
-list it in the configuration. If there is a plugin whose socket exists in the default location
-and you want to include it in the config for visibility, you can do so and omit the socket path - 
-this tell Synse Server to look for it in the default location.
-```yaml
-plugin:
-  unix:
-    foobar:
-```
-
-### Default Configuration
-The default configuration is as follows:
-```yaml
-locale: en_US
-pretty_json: false
-logging: info
-cache:
-  meta:
-    ttl: 20
-  transaction:
-    ttl: 300
-grpc:
-  timeout: 3
-```
+See the [API Documentation][synse-api] to learn what else Synse Server is capable of.
 
-### Configuration Options
+## Feedback
+Feedback for Synse Server or any component of the Synse ecosystem is greatly appreciated!
+If you experience any issues, find the documentation unclear, have requests for features,
+or just have questions about it, we'd love to know. Feel free to open an issue describing
```

Comment:
This last sentence may need cleanup.

Reply:
good catch. reading it again, it was some awkward phrasing. updated in latest commit.